pkg/build/strategies/sti/sti_test.go

```diff
@@ -808,6 +813,9 @@ func TestExecuteOK(t *testing.T) {
 		t.Errorf("PostExecutor not called with expected ID: %s",
 			pe.PostExecuteContainerID)
 	}
+	if !reflect.DeepEqual(ro.Env, expectedEnv) {
+		t.Errorf("Unexpected container environment passed to RunContainer: %v, should be %v", ro.Env, rh.config.Environment)
```

Comment:
Why we're comparing with `expectedEnv` but in the message showing `rh.config.Environment`? It's for readability or by mistake?